projects/ngx-charts-on-fhir/src/lib/data-layer-browser/data-layer-browser.component.css

```diff
@@ -2,6 +2,7 @@
   display: flex;
   flex-flow: column;
   height: 100vh;
+  padding-left: 2vh;
```

Comment:
why is padding-left based on the viewport height?

Reply:
@semicolin, try and run i have added this, we don't need this removing...